templates/spin_sidebar_v2.hbs

```diff
@@ -254,17 +254,10 @@
                         Kubernetes
                 </label>
                 <ul class="menu-list accordion-menu-item-content">
-                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/kubernetes" )}} class="active"
-                                        {{/if}} href="{{site.info.base_url}}/spin/v2/kubernetes">Spin on Kubernetes</a>
-                        </li>
-                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/kubernetes-sidecars" )}} class="active"
-                                        {{/if}} href="{{site.info.base_url}}/spin/v2/kubernetes-sidecars">Using Sidecars</a>
-                        </li>
-                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/kubernetes-dynamic-configuration" )}} class="active"
-                                        {{/if}} href="{{site.info.base_url}}/spin/v2/kubernetes-dynamic-configuration">Dynamic Application Configuration on Kubernetes</a>
-                        </li>
-                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/kubernetes-known-issues" )}} class="active"
-                                        {{/if}} href="{{site.info.base_url}}/spin/v2/kubernetes-known-issues">Known Issues</a>
+                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/spinkube" )}} class="active"
+                                        {{/if}} href="{{site.info.base_url}}/spin/v2/spinkube">SpinKube</a>
+                        <li><a {{#if (active_project request.spin-full-url "/spin/v2/spin-in-pods-legacy" )}} class="active"
+                                        {{/if}} href="{{site.info.base_url}}/spin/v2/spin-in-pods-legacy">Spin in Pods (Legacy)</a>
```

Comment:
Looking at these TOC entries reinforces my feeling about keeping "Spin in Kubernetes".  I look at this and go "which do I want?  I sure don't want some legacy thing but I don't know what SpinKube is, just give me a page that tells me what to do or what my options are."

But I may be in the minority on this _grin_

Reply:
Agreed: They will see `SpinKube` as soon as they click on the link anyway. This change helps the user choose. Nice catch @itowlson